Modules/ArcGIS/ArcGIS.psm1

```diff
@@ -388,12 +338,12 @@ function Get-ArcGISURL
         $RestURL = "https://$ServerExternalDNSName/$ServerContext/rest"
     }
     
-    if($PrimaryPortalMachine)
+    if($PrimaryServerMachineNode)
```

Comment:
Shouldn't this be
` if($PrimaryPortalMachineNode) `

?

Reply:
Yes, you are correct. Will fix this.